views/office.ejs

```diff
@@ -11,7 +11,7 @@
   <%- include('header'); %>
   <div class="matrix-content container-fluid">
     <!-- Modal -->
-    <div class="modal fade" id="exampleModalCenter" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
```

Comment:
I believe We have a good opportunity to change the modal Id from "exampleModalCenter" to "meetModal" or other more  cohesion name..

Reply:
@juliemar done in 4e6d82d1d889cae731d0395acb88be1763e717bf